.eslintrc.js

```diff
@@ -11,5 +11,5 @@ governing permissions and limitations under the License.
 */
 
 module.exports = {
-    "extends": "@adobe/eslint-config-asset-compute"
-};
\ No newline at end of file
+    "extends": "@adobe/eslint-config-asset-compute",
```

Comment:
Trailing comma?

Reply:
(Ignore that, putting it back and removing it was my change to retrigger tests).